uncompyle6/scanners/scanner2.py

```diff
@@ -271,7 +276,7 @@ def build_lines_data(self, co, n):
 
         # linestarts is a tuple of (offset, line number).
         # Turn that in a has that we can index
-        self.linestarts = list(findlinestarts(co))
+        self.linestarts = list(self.opc.findlinestarts(co))
```

Comment:
In my work for python 3.6 support in uncompyle I have done the same thing, however for me the python2 bytecode tests are failing when running on python3 since the unmarshalling code is converting what were strings in python2 to bytes (findlinestrings for python2 calls ord). In the previous version of xdis findlinestarts was used from the standard library so if you were running python3 it made sense to perform this step in the unmarhsalling. Now xdis uses findlinestarts based on the version of bytecode being decompiled. I think some change will be required either here or in the unmarshalling to get this working correctly, @rocky do you have any thoughts on this?